src/parser.jl

```diff
@@ -264,6 +264,7 @@ function parseexprs(lexed, debug=false)
         if length(thisarg) != 1
             println(stderr, "thisarg:")
             foreach(x->(print(stderr, "  "); sexpr(stderr, x); println(stderr)), thisarg)
+            print(length(thisarg))
```

Comment:
Intentional?

Reply:
Good catch! The block is useful but the line is definitely not needed. Will fix rn.